innovation_sweet_spots/pipeline/pilot/investment_predictions/create_dataset/utils.py

```diff
@@ -531,7 +496,7 @@ def add_n_months_before_first_investment_in_window(
     """
     cb_data["n_months_before_first_investment"] = n_months_delta(
         cb_data["first_funding_date_in_window"], cb_data["founded_on"]
-    )
+    ).apply(lambda x: x if x >= -1 else 0)
```

Comment:
Hi, just wanted to make sure that `null` cases will still be -1 rather than 0? (ie, if there hasn't been an investment)

Reply:
Yes they will be 🙂